lib/BirthdayCalendarGenerator.php

```diff
@@ -15,23 +17,19 @@ class BirthdayCalendarGenerator
 {
     /**
      * Input objects.
```

Comment:
defining array key/value brings a lot of benefits. in the next PR we maybe should increase the phpstan level, so these types are required

```suggestion
     * Input objects.
     *
     * @var list<Component\VCard>
```

but feel free to do these in a followup as this PR already is pretty big

Reply:
yes, I will followup with a next PR...